ui/raidboss/raidboss_options.ts

```diff
@@ -94,6 +94,16 @@ const defaultRaidbossConfigOptions = {
   InfoSoundVolume: 1,
   LongSoundVolume: 1,
   PullSoundVolume: 1,
+  RumbleEnabled: false,
+  InfoRumbleDuration: 400,
+  InfoRumbleWeak: 0.5,
+  InfoRumbleStrong: 0,
```

Comment:
Is there any information as to what a constitutes a 'weak' vs 'strong' rumble?
For instance, I'm surprised that any strong number is lower than a weak number because I would have assumed they're a spectrum or sine wave that the rumble goes through.

Reply:
"Strong" and "Weak" are general conventions. Gamepads can also just have two motors that have the same strength, at which point it's less "Strong"/"Weak" and more "Left"/"Right".

From the MDN documentation:
```
strongMagnitude: Rumble intensity of the low-frequency (strong) rumble motors, normalized to the range between 0.0 and 1.0.
weakMagnitude: Rumble intensity of the high-frequency (weak) rumble motors, normalized to the range between 0.0 and 1.0.
```

For instance, in most shooters, the "strong" motor will fire for explosions with a high magnitude, with the "weak" motor firing at low magnitude, whereas for gunfire the "weak" motor is used for most types of gun.